### ISSUE TITLE
Spellcheck

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#68]
```
There's a function that I miss: Spellcheking my tweets.
Is it possible?
Cheers
```

Original issue reported on code.google.com by `bicode...@gmail.com` on 3 Feb 2010 at 2:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 3 Feb 2010 at 3:59
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Issue 133 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 18 Mar 2010 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
Closed by 4a7e0dd476 revision.

(but I can't find the way to remember a last selected language)
```

Original comment by `tro...@gmail.com` on 26 Mar 2010 at 11:24
- Changed state: **Fixed**
